plugins/modules/net_tools/dnsmadeeasy.py

```diff
@@ -673,8 +678,11 @@ def main():
         # create record and monitor as the record does not exist
         if not current_record:
             record = DME.createRecord(DME.prepareRecord(new_record))
-            monitor = DME.updateMonitor(record['id'], DME.prepareMonitor(new_monitor))
-            module.exit_json(changed=True, result=dict(record=record, monitor=monitor))
+            if new_monitor['monitor'] and record_type == "A":
```

Comment:
The dictionary might be empty:
```suggestion
            if new_monitor.get('monitor') and record_type == "A":
```
Why is the `record_type == "A"` necessary?

Reply:
DNS Made Easy's monitoring and failover service is only available for A records.

More info: https://social.dnsmadeeasy.com/blog/failover-dns-made-easy-vs-constellix/